src/app/components/LimitIE9Comment/index.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+/*
+ * These, when used together, stop anything within them from loading on IE9 and below.
+ */
+
+export const IE9LimitOpen = () => (
+  <div dangerouslySetInnerHTML={{ __html: '<!--[if !IE]> -->' }} /> // eslint-disable-line react/no-danger
+);
+
+export const IE9LimitClose = () => (
+  <div dangerouslySetInnerHTML={{ __html: '<!-- <![endif]-->' }} /> // eslint-disable-line react/no-danger
```

Comment:
```suggestion
  <div dangerouslySetInnerHTML={{ __html: '<!--<![endif]-->' }} /> // eslint-disable-line react/no-danger
```

Reply:
Hmm this doesn't match `<!-- <![endif]-->` on https://www.quirksmode.org/css/condcom.html